rubin_jupyter_utils/hub/scanrepo/scanrepo.py

```diff
@@ -303,15 +289,13 @@ def get_all_tags(self):
         with start_action(action_type="get_all_tags"):
             return self._all_tags
 
-    def _get_url(self, **kwargs):
+    def _get_url(self, url, headers, **kwargs):
```

Comment:
Oh, huh.  I thought of a scanner object as operating on a fixed URL (that is, for a given repo you'd instantiate a new scanner).  This is fine though.